types/gapi.client.people/tslint.json

```diff
@@ -1,6 +1,8 @@
 {
     "extends": "dtslint/dt.json",
     "rules": {
-        "await-promise": false
+        "interface-name": [false],
```

Comment:
Please suppress this and the other two rules in the source instead of suppressing them globally. Thanks!

Reply:
Will do, thanks!